common/tier4_control_rviz_plugin/src/tools/manual_controller.cpp

```diff
@@ -133,8 +133,9 @@ void ManualController::update()
     const double eps = 0.001;
     if (ackermann.longitudinal.speed > eps) {
       gear_cmd.command = GearCommand::DRIVE;
-    } else if (ackermann.longitudinal.speed < -eps) {
+    } else if (cruise_velocity_ < eps && current_velocity_ < eps) {
```

Comment:
```suggestion
    } else if (ackermann.longitudinal.speed < -eps && current_velocity_ < eps) {
```

Reply:
@taikitanaka3 
`ackermann.longitudinal.speed < eps` is correct?

Reply:
-eps is correct but cruise velocity can be set 1.0km/h step so behavior is same as eps